sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_delete_events_v2/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+---
+friendly_name: FxA Delete Events
+description: Deletion events extracted from FxA auth server logs
+ used as signal for Mozilla to delete analysis data associated with
+ the user
+owners:
+  - kik@mozilla.com
+labels:
+  application: fxa
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_fxa_events
+  # This query references secret keys that are not available for dry runs,
+  # so we must explicitly write out dependencies. In this case, the query
+  # depends only on fxa logs produced via Stackdriver integration, so no other
```

Comment:
This was copied from the v1 ETL and I'm not sure if this Stackdriver part is actually true for the newer logs from GCP, but I don't have time at the moment to fix it up.  @akkomar or @kik-kik feel free to do so.

Reply:
Other than Stackdriver  was rebranded to Cloud Monitoring I think this comment is still true.

Reply:
```suggestion
  # depends only on fxa logs produced via Cloud Monitoring integration, so no other
```